docker/release/dockercomposefiles/docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
         soft: 65536 # maximum number of open files for the OpenSearch user, set to at least 65536 on modern systems
         hard: 65536
     volumes:
+      - opensearch-config:/usr/share/opensearch/config
```

Comment:
We shouldnt mount the config file version since users can choose to change them or not.
With mounting it is forcing the inode to be the same without the options to create/change to new inode.

Reply:
Not sure if I understood this correctly. The reason for creating a volume mount here is to create a persistent storage for generated certs that helps re-using them across all containers, as well as make the configs agnostic to container restarts.
If a user wants to mount a custom config, they still can do so by binding custom yml to docker like 
```./custom-opensearch.yml:/usr/share/opensearch/config/opensearch.yml```

Reply:
I still think we should leave the config file dir as it is and let the user decide the mount themselves.
If this file bound to change you also need to update the one on the website download page.

Reply:
I think I understood what you're trying to say. Let me clarify - mounting is not forcing users to use the mounted config, it is only providing a secure default option. Users can always override the default mount with a their custom config by making changes in `docker-compose.yml` (like they do today).  

Another option was to put certs in a different folder under config and mount only that folder instead of config/, but in that case `opensearch.yml` would still have references to those certs so if a user is using a custom-opensearch.yml, they would still need to override the certs mount. Another reason to not go with this option is changing certs path might break backward compatibility, and since this is intended for 1.x, that would not be desirable.

Reply:
I have changed this further. Instead of mounting the entire config, now it is just mounting the certs/. that way users can still override the `opensearch.yml` with a custom one, in the same manner as they do today.